api/app_analytics/constants.py

```diff
@@ -0,0 +1,6 @@
+ANALYTICS_READ_BUCKET_SIZE = 15
+
+# get_usage_data() related period constants
+CURRENT_BILLING_PERIOD = "current_billing_period"
+PREVIOUS_BILLING_PERIOD = "previous_billing_period"
+_90_DAY_PERIOD = "90_day_period"
```

Comment:
Why is this prefixed with underscore (ergo, private)?

Reply:
Because python (very annoyingly) doesn't allow variables or constants to start with a number.

Reply:
I also had this question, so I think it warrants a better resolution tbh. Why not `NINETY_DAY_PERIOD` ? If we're adamant on keeping the leading underscore for whatever reason, lets at least add a comment here.

Reply:
Great solution Matt, I agree

Reply:
Ok this is done.